.travis.yml

```diff
@@ -137,7 +137,7 @@ jobs:
       name: "macOS: clang"
       os: osx
       compiler: clang
-      osx_image: xcode10.2  # macOS 10.14
+      osx_image: xcode11.3  # macOS 10.14
```

Comment:
if this works, should also revert the `update: true` workaround I pushed to master.

Reply:
Did not work :(